mne_bids/write.py

```diff
@@ -1215,11 +1216,15 @@ def write_raw_bids(raw, bids_path, events_data=None,
 
     # for MEG, we only write coordinate system
     if bids_path.datatype == 'meg' and not emptyroom:
+        # note that any coordsystem.json file shared within sessions
+        # will be the same across all runs (currently). So
+        # overwrite is set to True always
+        # XXX: improve later when BIDS is updated
         _coordsystem_json(raw=raw, unit=unit, hpi_coord_system=orient,
                           sensor_coord_system=orient,
                           fname=coordsystem_path.fpath,
                           datatype=bids_path.datatype,
-                          overwrite=overwrite, verbose=verbose)
+                          overwrite=True, verbose=verbose)
```

Comment:
In the related issue discussion, we said that we should fail hard if `coordsystem.json` already exists and its content doesn't match what we would be going to write for the current run